<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="acs-PC" name="testcase.TC001_Create_an_Account" tests="5" failures="0" timestamp="30 Sep 2019 06:42:27 GMT" time="99.219" errors="5">
  <testcase name="TC001_Create_An_Account" time="17.780" classname="testcase.TC001_Create_an_Account">
    <error type="java.io.FileNotFoundException" message="C:\Eclipse_Data\Maven_Project\src\test\resources\screenShots\screnshot_2019.09.30.12.12.02.png (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Eclipse_Data\Maven_Project\src\test\resources\screenShots\screnshot_2019.09.30.12.12.02.png (The system cannot find the path specified)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.openqa.selenium.io.FileHandler.copyFile(FileHandler.java:138)
	at org.openqa.selenium.io.FileHandler.copy(FileHandler.java:117)
	at testbase.TestBase.takescreenshots(TestBase.java:104)
	at commonmethods.Commonmethod.clickOnWebElement(Commonmethod.java:27)
	at pages.LogInpage.createAnAccount(LogInpage.java:12)
	at testcase.TC001_Create_an_Account.TC001_Create_An_Account(TC001_Create_an_Account.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:663)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:849)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1200)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1125)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- TC001_Create_An_Account -->
  <testcase name="TC001_Create_An_Account" time="24.478" classname="testcase.TC001_Create_an_Account">
    <error type="java.io.FileNotFoundException" message="C:\Eclipse_Data\Maven_Project\src\test\resources\screenShots\screnshot_2019.09.30.12.11.09.png (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Eclipse_Data\Maven_Project\src\test\resources\screenShots\screnshot_2019.09.30.12.11.09.png (The system cannot find the path specified)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.openqa.selenium.io.FileHandler.copyFile(FileHandler.java:138)
	at org.openqa.selenium.io.FileHandler.copy(FileHandler.java:117)
	at testbase.TestBase.takescreenshots(TestBase.java:104)
	at commonmethods.Commonmethod.clickOnWebElement(Commonmethod.java:27)
	at pages.LogInpage.createAnAccount(LogInpage.java:12)
	at testcase.TC001_Create_an_Account.TC001_Create_An_Account(TC001_Create_an_Account.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:663)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:849)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1200)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1125)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- TC001_Create_An_Account -->
  <testcase name="TC001_Create_An_Account" time="17.619" classname="testcase.TC001_Create_an_Account">
    <error type="java.io.FileNotFoundException" message="C:\Eclipse_Data\Maven_Project\src\test\resources\screenShots\screnshot_2019.09.30.12.12.22.png (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Eclipse_Data\Maven_Project\src\test\resources\screenShots\screnshot_2019.09.30.12.12.22.png (The system cannot find the path specified)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.openqa.selenium.io.FileHandler.copyFile(FileHandler.java:138)
	at org.openqa.selenium.io.FileHandler.copy(FileHandler.java:117)
	at testbase.TestBase.takescreenshots(TestBase.java:104)
	at commonmethods.Commonmethod.clickOnWebElement(Commonmethod.java:27)
	at pages.LogInpage.createAnAccount(LogInpage.java:12)
	at testcase.TC001_Create_an_Account.TC001_Create_An_Account(TC001_Create_an_Account.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:663)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:849)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1200)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1125)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- TC001_Create_An_Account -->
  <testcase name="TC001_Create_An_Account" time="19.786" classname="testcase.TC001_Create_an_Account">
    <error type="java.io.FileNotFoundException" message="C:\Eclipse_Data\Maven_Project\src\test\resources\screenShots\screnshot_2019.09.30.12.11.42.png (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Eclipse_Data\Maven_Project\src\test\resources\screenShots\screnshot_2019.09.30.12.11.42.png (The system cannot find the path specified)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.openqa.selenium.io.FileHandler.copyFile(FileHandler.java:138)
	at org.openqa.selenium.io.FileHandler.copy(FileHandler.java:117)
	at testbase.TestBase.takescreenshots(TestBase.java:104)
	at commonmethods.Commonmethod.clickOnWebElement(Commonmethod.java:27)
	at pages.LogInpage.createAnAccount(LogInpage.java:12)
	at testcase.TC001_Create_an_Account.TC001_Create_An_Account(TC001_Create_an_Account.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:663)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:849)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1200)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1125)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- TC001_Create_An_Account -->
  <testcase name="TC001_Create_An_Account" time="19.556" classname="testcase.TC001_Create_an_Account">
    <error type="java.io.FileNotFoundException" message="C:\Eclipse_Data\Maven_Project\src\test\resources\screenShots\screnshot_2019.09.30.12.11.27.png (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Eclipse_Data\Maven_Project\src\test\resources\screenShots\screnshot_2019.09.30.12.11.27.png (The system cannot find the path specified)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at org.openqa.selenium.io.FileHandler.copyFile(FileHandler.java:138)
	at org.openqa.selenium.io.FileHandler.copy(FileHandler.java:117)
	at testbase.TestBase.takescreenshots(TestBase.java:104)
	at commonmethods.Commonmethod.clickOnWebElement(Commonmethod.java:27)
	at pages.LogInpage.createAnAccount(LogInpage.java:12)
	at testcase.TC001_Create_an_Account.TC001_Create_An_Account(TC001_Create_an_Account.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:663)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:849)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1200)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1125)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- TC001_Create_An_Account -->
</testsuite> <!-- testcase.TC001_Create_an_Account -->
